libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/CodegenLayout.kt

```diff
@@ -147,14 +147,26 @@ internal abstract class CodegenLayout(
     return "${name.capitalizeFirstLetter()}Builder"
   }
 
-  fun mapName(name: String): String {
+  fun unknownBuilderName(name: String): String {
```

Comment:
```suggestion
  fun otherBuilderName(name: String): String {
```

Reply:
Actually, same remark as just below. Just use `builderName` and keep everything symmetrical?